src/backend/columnar/columnar_tableam.c

```diff
@@ -1809,3 +1805,95 @@ alter_columnar_table_reset(PG_FUNCTION_ARGS)
 
 	PG_RETURN_VOID();
 }
+
+
+/*
+ * columnar_metapage - UDF to print the contents of metapage, if it
+ * exists, otherwise return NULL.
+ */
+PG_FUNCTION_INFO_V1(columnar_metapage);
+Datum
+columnar_metapage(PG_FUNCTION_ARGS)
```

Comment:
what about defining this in `columnar_debug.c` ?

Reply:
Done.